nvtabular/columns/schema.py

```diff
@@ -14,7 +14,101 @@
 # limitations under the License.
 #
 from dataclasses import dataclass, field
-from typing import List, Optional, Text
+from pathlib import Path
+from typing import Dict, List, Optional, Text
+
+import numpy
+from google.protobuf import json_format, text_format
```

Comment:
We need to add something like 
https://github.com/NVIDIA/NVTabular/blob/e9c0795ab68d4d42b13d3a73e888c73b1483fb73/nvtabular/inference/triton/__init__.py#L25-L26

before any of these protobuf imports . I believe this is the root cause of the cpu CI failures you're seeing

Reply:
roger will fix now